gpii/node_modules/flowManager/configs/gpii.flowManager.config.local.base.json5

```diff
@@ -2,13 +2,30 @@
     "type": "gpii.flowManager.config.local.base",
     "options": {
         "distributeOptions": {
+            "kettleServer.ws": {
+                "record": "kettle.server.ws",
+                "target": "{that server}.options.gradeNames"
+            },
             "flowManager.local": {
-                "record": "gpii.flowManager.local",
+                "record": ["gpii.flowManager.local", "gpii.flowManager.ws", "gpii.contextManager.app"],
```

Comment:
I think we had also planned to eliminate the grade gpii.flowManager.ws - there's no point separating the server in two based on the hosting technology. This means the options distribution above can be destroyed also